### ISSUE TITLE
[Tool bar][Addon] More toolbar

### ISSUE THREAD

**Issue by GunGunGun** [tablacus/TablacusExplorer#722]
The toolbar addon currently only allow 1 single toolbar, it would be greate to be able to addon more toolbar to top, bottom, left, right and more like Directory Opus.
https://tablacus.github.io/wiki/addons/toolbar.html

**Comment by ccfs** [comment]
The [patched version](https://github.com/tablacus/TablacusExplorer/issues/692) now supports multiple tool bars.

**Comment by Leo9idas** [comment]
Hello and thank you for your amazing file explorer. One question. Wanting to use the patched version with the more toolbars and the other enhancements but after patching the other addons ask for program update. If I update  I lose all the patched enhancements and I have errors. Is this something to be expected ? Thank you in advance !

**Comment by ccfs** [comment]
Yes, if you update the patched version through the official channel (i.e. Help > Check for updates) TE will get errors. But you can run patch.cmd again to re-apply.

I have upgraded the base version of the patch to 24.2.27 on March 28. Make sure you have got the same version before you install the latest patch.
